app/test_split_integer.py

```diff
@@ -2,20 +2,22 @@
 
 
 def test_sum_of_the_parts_should_be_equal_to_value() -> None:
-    pass
+    assert (sum(split_integer(6, 2)) == 6), \
```

Comment:
The parentheses around the assert condition are unnecessary. You can write `assert sum(split_integer(6, 2)) == 6` instead.